Module.cs

```diff
@@ -128,7 +128,22 @@ private async Task LoadRoles()
                 {
                     try
                     {
-                        role.Icon = ContentsManager.GetTexture(role.IconPath);
+                        if (role.IconPath.StartsWith("icons"))
+                        {
+                            role.Icon = ContentsManager.GetTexture(role.IconPath);
+                        }
+                        else
+                        {
+                            if (!File.Exists(role.IconPath)) return;
+                            using (var textureStream = File.Open(role.IconPath, FileMode.Open))
```

Comment:
Nitpick: you can inline the `using` by adding a semi-colon at the end, then omitting the braces, saving us from a nesting level!